test/publishing/dependency-check.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var dependencies = require('../../package.json').dependencies;
+var dependencyNames = Object.keys(dependencies);
+var packages = dependencyNames.filter(function (d) {
+  // Remove scoped packages
+  return d[0] !== '@';
```

Comment:
Why do we take out the scoped package? That's one of the core shared dependencies.